spec/system/item_system_spec.rb

```diff
@@ -76,7 +76,7 @@
   end
 
   describe "destroying items" do
-    subject { create(:item, name: "DELETEME", organization: @user.organization) }
```

Comment:
Was this an intended to be in the diff here?

Reply:
Rubocop :roll_eyes:

Reply:
Actually yeah -- this was intentional. The test was failing because the element wasn't appearing in the viewport of the browser driver. 

Prefixing with "AAA" ensured it would be at the top. There are still a few other failing specs for the same reason but they weren't as easy to address.